### DISCUSSION TITLE
Can I use CI/CD with `serverless.ts` file instead of `serverless.yml`?

### DISCUSSION THREAD

**Discussion by andrew-sol** [serverless/serverless#12267] [Q&A]
I connected my GitHub repo to the CI/CD and it says "Invalid serverless configuration path. Please try a different path.".

Under the "Respository" section I see the following label: "Select a repository and configure build settings. Make sure your repository has a serverless.yml file within it."

Does it mean that it's not possible to use the tool without the `serverless.yml` file? AFAIK there's no way of generating the yml config without resolving variables.

**Comment by edercarloscosta** [comment]
Hi @andrew-sol 

I think you are facing this https://stackoverflow.com/questions/66726549/how-to-convert-my-serverless-ts-file-to-serverless-yaml-file-and-make-the-server
